proto/uber/cadence/admin/v1/service.proto

```diff
@@ -121,6 +121,18 @@ service AdminAPI {
 
   // MaintainCorruptWorkflow deletes a workflow if its history is corrupt due to underlying DB issues (e.g. Cassandra resurrections)
   rpc MaintainCorruptWorkflow(MaintainCorruptWorkflowRequest) returns (MaintainCorruptWorkflowResponse);
+
+  // GetIsolationGroups
+  rpc GetGlobalIsolationGroups(GetGlobalIsolationGroupsRequest) returns (GetGlobalIsolationGroupsResponse);
```

Comment:
Who will call the `Get` APIs?

Reply:
CLI / SRE tooling. I doubt it's possible to have such an endpoint to add drains without being able to read also